src/state/routing/slice.ts

```diff
@@ -119,11 +127,10 @@ export const routingApi = createApi({
         )
       },
       async queryFn(args, _api, _extraOptions, fetch) {
-        const { tokenInAddress, tokenInChainId, tokenOutAddress, tokenOutChainId, amount, routerPreference, type } =
-          args
-
-        try {
-          if (routerPreference === RouterPreference.API || routerPreference === RouterPreference.AUTO) {
+        if (args.routerPreference === RouterPreference.API || args.routerPreference === RouterPreference.AUTO) {
```

Comment:
What about `INTERNAL_ROUTER_PREFERENCE_PRICE`?
What codepath is that meant to use?